package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-auth-js",
   "description": "The Okta Auth SDK",
-  "version": "5.4.1",
+  "version": "5.4.2",
```

Comment:
Why is version 5.4.1 in master? For a patch release this PR should be against 5.4 branch

Reply:
Looks like the branch may be based on 5.4 but the PR is based on master. please fix.

Reply:
good catch, actually I don't know why... 
This PR looks correct  https://github.com/okta/okta-auth-js/pull/919/files

Reply:
OK, just the PR issue, changed.

Reply:
It is correct in master: https://github.com/okta/okta-auth-js/blob/master/package.json#L5